zrml/prediction-markets/src/lib.rs

```diff
@@ -586,13 +636,12 @@ mod pallet {
             ensure_signed(origin.clone())?;
             let weight_bcs = Self::buy_complete_set(origin.clone(), market_id, amount)?
```

Comment:
`buy_complete_set` still uses `calculate_actual_weight`, which means that `UnexpectedNoneInPostInfo` might actually be thrown here. Let's get rid of `calculate_actual_weight` alltogether.

Reply:
I've done it 🥵 69417b152632f6eb6b50092b133ab1b484cd1e25 Thanks for your support.